### DISCUSSION TITLE
Navigation Menu positioning the viewport

### DISCUSSION THREAD

**Discussion by KrokoNinja** [shadcn-ui/ui#2897] [Q&A]
Hey everybody,
i would like to position the viewport (which i think is the one who gives me the menu content am i right?) under the trigger. 
So when i hover the fisrt it should be left aligned under the first, when i hover the second it should be aligned under the second and so on. 
Is this possible? :) 
Thank you for your answers and ideas

**Comment by lloydrichards** [comment]
there is an issue related to this (#418) where some people suggest ways to adapt the component, but the real solution would be exposing the position of the trigger like [this issue ](https://github.com/radix-ui/primitives/issues/1462)in radix/ui suggests

**Comment by limehawk** [comment]
Checkout my comment here: https://github.com/radix-ui/primitives/issues/1462#issuecomment-3724992120
